src/ed.cpp

```diff
@@ -239,8 +189,8 @@ bool srvUpdate(ed_msgs::UpdateSrv::Request& req, ed_msgs::UpdateSrv::Response& r
 
 bool srvQuery(ed_msgs::Query::Request& req, ed_msgs::Query::Response& res)
 {
-    tue::Timer timer;
-    timer.start();
+//    tue::Timer timer;
+//    timer.start();
```

Comment:
remove these comments (or is there a policy to leave these?)

Reply:
Done